src/Core/Services/TypeHelper.cs

```diff
@@ -227,7 +227,11 @@ public static JsonDataType GetJsonDataTypeFromSystemType(Type type)
         /// <exception>Failed type conversion.</exception>"
         public static Type GetSystemTypeFromSqlDbType(string sqlDbTypeName)
         {
-            if (Enum.TryParse(sqlDbTypeName, ignoreCase: true, out SqlDbType sqlDbType))
+            // Remove the length specifier from the type name if it exists.Example: varchar(50) -> varchar
+            int separatorIndex = sqlDbTypeName.IndexOf('(');
```

Comment:
How was this identified? 

In our .sql script that we have, there are few columns that have a type varchar(some_size), it seems like without this change, it should've errored out long time back?

Reply:
This is a change Rohan introduced for DW: https://github.com/Azure/data-api-builder/pull/2008/files#diff-f661500d3d243d627951f0d434d5241ec0f00b3d3798061a0733362c16f761ce

Reply:
Updated the branch.